external/loaders/loaders/_one_ahead.py

```diff
@@ -52,4 +52,3 @@ def __del__(self):
         # check necessary in case exceptions occur before this is defined
         if hasattr(self, "_executor"):
             self._executor.shutdown(wait=True)
-        super().__del__()
```

Comment:
IIRC this gave me an error that the superclass has no del defined, which only shows up when python exits but is still an exception.